server/built_in_tools.go

```diff
@@ -391,6 +391,79 @@ func (p *Plugin) toolSearchServer(llmContext *llm.Context, argsGetter llm.ToolAr
 	return formatted, nil
 }
 
+/*type CreateChannelArgs struct {
```

Comment:
Why this is commented out? Makes sense to directly remove this?

Reply:
removed.